content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -0,0 +1,202 @@
+# Sharing Experiments
+
+There are two types of remotes that can store experiments. Git remotes are
+distributed copies of the Git repository, for example on GitHub or GitLab.
+
+[DVC remotes](/doc/command-reference/remote) on the other hand are
+storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
+configure with `dvc remote`. DVC uses them to store and fetch large files that
+don't normally fit inside Git repos.
+
+DVC needs both kinds of remotes for backing up and sharing experiments.
+
+Experiment files that are normally tracked in Git (like code versions) are
+shared using Git remotes, and files or directories tracked with DVC (like
+datasets) are shared using DVC remotes.
+
+> See [Git remotes guide] and `dvc remote add` for information on setting them
+> up.
+
+[git remotes guide]:
+  https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+
+Normally, there should already be a Git remote called `origin` when you clone a
+repo. Use `git remote -v` to list your Git remotes:
+
+```dvc
+$ git remote -v
+origin  https://github.com/iterative/get-started-experiments (fetch)
+origin  https://github.com/iterative/get-started-experiments (push)
+```
+
+Similarly, you can see the DVC remotes in you project using `dvc remote list`:
+
+```dvc
+$ dvc remote list
+storage https://remote.dvc.org/get-started-experiments
+```
+
+_Q: What about neither, `cache: false` objects, and objects tracked both by DVC
+and Git?_
+
+## Uploading experiments to remotes
+
+You can upload an experiment and its files to both remotes using `dvc exp push`
+(requires the Git remote name and experiment name as arguments).
+
+```dvc
+$ dvc exp push origin exp-abc123
+```
+
+> Use `dvc exp show` to find experiment names.
+
+This pushes the necessary DVC-tracked files from the cache to the default DVC
+remote (similar to `dvc push`). You can prevent this behavior by using the
+`--no-cache` option to the command above.
+
+If there's no default DVC remote, it will ask you to define one with
+`dvc remote default`. If you don't want a default remote, or if you want to use
+a different remote, you can specify one with the `--remote` (`-r`) option.
+
+DVC can use multiple threads to upload files (4 per CPU core by default). You
+can set the number with `--jobs` (`-j`). Please note that increases in
+performance also depend on the connection bandwidth and remote configurations.
+
+DVC has a mechanism called the [run-cache] that stores the artifacts from
+intermediate stages. For example, if there is an intermediate step that applies
+data-augmentation on your dataset and you would like to push these artifacts as
+well as the end products of the experiments, you can use `--run-cache` flag to
+push all of these to the DVC remote. `--run-cache` flag pushes all artifacts
+referenced in `dvc.lock` file.
+
+[run-cache]: /doc/user-guide/project-structure/internal-files#run-cache
```

Comment:
I don't think that's what the run-cache does. Intermediate artifacts of the experiment should be moved by `exp push/pull`. The run-cache probably only exists here as an edge case when you use `run/repro` in between `exp run`s. If so, no nee to even mention it probably.

Reply:
> p.s. This is the only sort of major thing I would prefer to address before merge.

Reply:
Oh and if we need to keep this part about the run-cache, what about `pull --run-cache`? It's not mentioned in that section

Reply:
Frankly I didn't test `--run-cache` option and this is what I got from the Ref. Need to test this I believe.

Reply:
For now I would remove these run-cache mentions probably.

Reply:
Removed for now.

Reply:
Well, I left a small note.